### DISCUSSION TITLE
更新到最新版之后，Firefox 浏览器自动填充账密功能失效了

### DISCUSSION THREAD

**Discussion by mx2913** [1Panel-dev/1Panel#5411] [Q&A]
### 1Panel 版本

社区版v1.10.10-lts

### 问题描述

更新到最新版之后，Firefox 浏览器自动填充账密功能失效了，需要手动点按用户输入框选择

### 重现步骤

旧版本是支持直接填充的

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，刚在本地环境测试未复现上述问题。
> 可以提供一下录屏

**Reply by mx2913** [reply]
有旧版本安装的命令吗，我安装个旧版本录个屏对比下

**Reply by wanghe-fit2cloud** [reply]
[安装包下载地址](https://1panel.cn/docs/installation/package_installation/) ，提供了v1.10.9 和 v1.10.10 两个版本。

下图是我在本地环境尝试 Firefox 登录：
![image](https://github.com/1Panel-dev/1Panel/assets/46734380/5b352eac-d74b-485e-a70e-bbab02f32eb0)

**Reply by mx2913** [reply]
我本地试了 `1.10.8` 到 `1.10.10` 三个版本，都没自动填充

<img width="1349" alt="图片" src="https://github.com/1Panel-dev/1Panel/assets/15864713/f6e56a85-4f7c-484c-b27c-38f40b8f28f6">

上图中的是服务器上用的最新版本，本地(下图)的也是同样情况，怀疑有可能不是 `https` 访问的影响，正在弄个临时服务器尝试 `https` 能否自动填充
<img width="1319" alt="图片" src="https://github.com/1Panel-dev/1Panel/assets/15864713/b6640af4-c805-4e10-b358-8324f0d521a3">

**Reply by QYG2297248353** [reply]
有的浏览器要输入一个字母，或者鼠标激活用户名或密码输入框才行。对于本地地址 127.0.0.1，192.168.0.0 之类的用不了，但是公网ip又可以使用，

**Reply by mx2913** [reply]
> 有的浏览器要输入一个字母，或者鼠标激活用户名或密码输入框才行。对于本地地址 127.0.0.1，192.168.0.0 之类的用不了，但是公网ip又可以使用，

我一直用的 Firefox，之前一直是可以的，后面更新了最新版突然发现要手动了

**Reply by mx2913** [reply]
Firefox 的版本信息
![图片](https://github.com/1Panel-dev/1Panel/assets/15864713/22a14fb0-990f-4ad0-9ea3-11f4ad9b419d)

**Reply by mx2913** [reply]
录频文件 @wanghe-fit2cloud 

https://github.com/1Panel-dev/1Panel/assets/15864713/58aba3a0-44bc-4005-b6be-467cb233f6b2
